src/Symfony/Component/Mime/Tests/Header/DateHeaderTest.php

```diff
@@ -44,7 +44,7 @@ public function testDateTimeIsConvertedToImmutable()
 
     public function testDateTimeIsImmutable()
     {
-        $header = new DateHeader('Date', $dateTime = new \DateTime('2000-01-01 12:00:00 Europe/Berlin'));
+        $header = new DateHeader('Date', $dateTime = new \DateTimeImmutable('2000-01-01 12:00:00 Europe/Berlin'));
```

Comment:
This defeats the purpose of the test, as `$dateTime->setDate` on next line would not mutate the object passed to the DateHeader anymore. This test is also about covering what happens when passing a mutable DateTime as argument, to ensure we convert it to DateTimeImmutable early enough.